core/graph/src/main/java/uk/gov/gchq/gaffer/graph/hook/FunctionAuthoriser.java

```diff
@@ -71,7 +71,7 @@ public void preExecute(final OperationChain<?> opChain, final Context context) {
             // This should never happen in real life as operation chains should
             // always be json serialisable. However this could happen if using a
             // mock in testing. To account for this, it will be logged.
-            LOGGER.warn("Failed to serialise operation chain: " + opChain + " due to " + e.getMessage());
+            LOGGER.warn("Failed to serialise operation chain: {} due to {}", opChain , e.getMessage());
```

Comment:
Your PR is failing the checkstyle stage of the CI, mostly due to whitespace issues around commas. You can run the checkstyle locally to see what exactly is wrong with `mvn checkstyle:checkstyle`.

Reply:
Will fix this. Thanks for the notice.